xbmc/filesystem/VideoDatabaseDirectory/DirectoryNode.cpp

```diff
@@ -44,6 +44,7 @@
 #include "guilib/LocalizeStrings.h"
 #include "utils/Variant.h"
 #include "video/VideoDatabase.h"
+#include "settings/settings.h"
```

Comment:
same here missed an uppercase S here, should be settings/Settings.h and that's why it's failing on linux/android